test/test_trixi.jl

```diff
@@ -0,0 +1,238 @@
+using Test: @test
+import Trixi
+
+# Use a macro to avoid world age issues when defining new initial conditions etc.
+# inside an elixir.
+"""
+    @test_trixi_include(elixir; l2=nothing, linf=nothing,
+                                atol=500*eps(), rtol=sqrt(eps()),
+                                parameters...)
+
+Test Trixi by calling `trixi_include(elixir; parameters...)`.
+By default, only the absence of error output is checked.
+If `l2` or `linf` are specified, in addition the resulting L2/Linf errors
+are compared approximately against these reference values, using `atol, rtol`
+as absolute/relative tolerance.
+"""
+macro test_trixi_include(elixir, args...)
+    local l2 = get_kwarg(args, :l2, nothing)
+    local linf = get_kwarg(args, :linf, nothing)
+    local atol = get_kwarg(args, :atol, 500 * eps())
+    local rtol = get_kwarg(args, :rtol, sqrt(eps()))
+    local skip_coverage = get_kwarg(args, :skip_coverage, false)
+    local coverage_override = expr_to_named_tuple(get_kwarg(args, :coverage_override, :()))
+    if !(:maxiters in keys(coverage_override))
+        # maxiters in coverage_override defaults to 1
+        coverage_override = (; coverage_override..., maxiters = 1)
```

Comment:
I guess that's why the simulation stops after one time step. Have you tried to include something like https://github.com/trixi-framework/Trixi.jl/blob/main/test/runtests.jl#L46 in your `runtests.jl`?